ui/src/logs/components/LogsTable.tsx

```diff
@@ -1,39 +1,64 @@
 import _ from 'lodash'
 import moment from 'moment'
-import React, {PureComponent, MouseEvent} from 'react'
+import React, {Component, MouseEvent} from 'react'
 import {Grid, AutoSizer} from 'react-virtualized'
 import {getDeep} from 'src/utils/wrappers'
 import FancyScrollbar from 'src/shared/components/FancyScrollbar'
 
-const ROW_HEIGHT = 40
+const ROW_HEIGHT = 30
+const HIGHLIGHT_COLOR = '#555'
 
 interface Props {
   data: {
     columns: string[]
     values: string[]
   }
+  scrolledToTop: boolean
```

Comment:
there was talk about prefixing boolean variables with `is`, how do you feel about making that change here?

Reply:
👍